Source/Core/DolphinQt2/QtUtils/QInputDialog.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/QtUtils/QInputDialog.h"
+
+#include "Core/HotkeyManager.h"
+
+QInputDialog::QInputDialog(QWidget* parent) : QDialog(parent)
```

Comment:
I think I can name a few: 
* Frameadvancing while working within the Debugger (obv. not implemented in Qt yet)
* Pausing / Unpausing the game while changing settings
(Probably quite a few more...)